libraries/AP_HAL_PX4/Util.cpp

```diff
@@ -183,6 +183,9 @@ void PX4Util::perf_count(perf_counter_t h)
 void PX4Util::set_imu_temp(float current)
 {
     if (!_heater.target || *_heater.target == -1) {
+#if defined(CONFIG_ARCH_BOARD_PX4FMU_V3)
```

Comment:
They should install V3 on a Cube, but your right, because V2 flashes onto a cube we should definitely be checking if BRD_TYPE == 3 as well. I can see the argument for always doing it on a V3 build, but I'd say a V2 build should still check BRD_TYPE just in case... (Also raises the point that on somethings we'd rather err on our idea of what board it is rather then the users).